contracts/cw-native-staked-balance-voting/src/msg.rs

```diff
@@ -4,10 +4,22 @@ use cw_utils::Duration;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, JsonSchema, Debug, Clone)]
+#[serde(rename_all = "snake_case")]
+pub enum Owner {
```

Comment:
I think we should move this to a package soon

Reply:
Would be good to have a look through our code and find more things we should put into packages at some point.